src/Core/Style.js

```diff
@@ -345,6 +350,10 @@ class Style {
         defineStyleProperty(this, 'point', 'radius', params.point.radius, 2.0);
         defineStyleProperty(this, 'point', 'width', params.point.width, 0.0);
         defineStyleProperty(this, 'point', 'base_altitude', params.point.base_altitude, base_altitudeDefault);
+        defineStyleProperty(this, 'point', 'model', params.point.model);
+
+        this.light = {};
+        defineStyleProperty(this, 'light', 'object', params.light.object);
```

Comment:
Why would we add light to itowns Style ? I might be missing something, but I think this should be removed from the commit of this PR as it is not within its scope.